### DISCUSSION TITLE
请问是否有overleaf上的模板？

### DISCUSSION THREAD

**Discussion by KGoinger** [mengchaoheng/SCUT_thesis#32] [Q&A] [answered]
请问是否有overleaf上的模板？谢谢！

**Comment by mengchaoheng** [comment]
他只是一个在线编辑器，你把文件从GitHub下载之后放到上面去运行应该不是很难。您试试？

**Reply by KGoinger** [reply]
您好，之前有尝试过直接上传项目到overleaf，但是发现在overleaf上无法编译，会报很多错误，怀疑可能是编译链的问题。

**Reply by mengchaoheng** [reply]
不可能编译不了的，首先你要上传万所有文件，其次你要设置正确的编译器和参考文献工具，然后就可以了

**Reply by mengchaoheng** [reply]
现在给你试试，等一下

**Comment by mengchaoheng** [comment]
> 请问是否有overleaf上的模板？谢谢！

GitHub私有库用起来比overleaf，也不差

**Comment by mengchaoheng** [comment]
考虑到它确实省去了部署环境的麻烦，我有空添加支持

**Comment by mengchaoheng** [comment, marked as answer]
已经更新，确保自己的论文在自己的分支上了，可以git pull然后git checkout overleaf分支得到最新的模版可以在overleaf使用，当然也可以直接去overleaf找。

**Reply by mengchaoheng** [reply]
@KGoinger

**Reply by mengchaoheng** [reply]
overleaf还没审核通过，你看看readme怎么获得支持overleaf的模版，自己上传创建即可。@KGoinger

**Reply by KGoinger** [reply]
非常感谢您！我尝试一下。

**Reply by KGoinger** [reply]
按照您的方法已经成功了：直接切换到overleaf分支下载代码，再上传overleaf并把编译器改为XeLaTeX就可以了。谢谢！
